### ISSUE TITLE
Add "make pot" task to Gulp

### ISSUE THREAD

**Issue by dannyvankooten** [ibericode/mailchimp-top-bar#59]
After #54, we should have a way to keep the `.pot` file updated at all times. `gulp-wp-pot` is great for this.

```js
const wpPot = require('gulp-wp-pot');
const sort = require('gulp-sort');

gulp.task('languages', function () {
    return gulp.src('src/**/*.php')
        .pipe(sort())
        .pipe(wpPot( {
            domain: 'mailchimp-top-bar',
            destFile:'mailchimp-top-bar.pot'
        } ))
        .pipe(gulp.dest('languages'));
});
```

**Comment by dannyvankooten** [comment]
Closed by 9c45c987db90dc2a187e56eb36eef09a1f705301

**Comment by shakaran** [comment]
@dannyvankooten if it is useful for you (more extended, but a pretty more old), you can see this old source that I write for this with more advanced options, not sure if it is updated for today use

http://bazaar.launchpad.net/~shakaran/tivion/tivion/view/head:/scripts/make_pot.sh
